sdk/storage/azure-storage-files-datalake/src/datalake_file_client.cpp

```diff
@@ -335,7 +335,10 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     ret.TransactionalContentHash = std::move(result->TransactionalContentHash);
     ret.ETag = std::move(result->ETag);
     ret.LastModified = std::move(result->LastModified);
-    ret.LeaseDuration = std::move(result->LeaseDuration);
+    // FIXME
+    ret.LeaseDuration = result->LeaseDuration.GetValue().count() == -1
```

Comment:
If I were you, to be on a safe side, I would consider to check for `< 0`, instead of `== -1`. Here, and in other places. Possibly. I don't know the entire context.

Reply:
This is temporary code, @katmsft will remove it soon.